### PR TITLE
Enhance push csv script to support update csv

### PR DESCRIPTION
**Pull request by DanielXLee**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: common/scripts/push-csv.sh [hunk 1]
```diff
@@ -47,13 +47,26 @@ else
...
+# Delete application release in repository
+function delete_csv() {
+  echo "Delete release ${RELEASE} of package ${QUAY_REPOSITORY} from namespace ${QUAY_NAMESPACE}"
+  curl -H "Content-Type: application/json" \
+      -H "Authorization: ${AUTH_TOKEN}" \
+      -XDELETE https://quay.io/cnr/api/v1/packages/"${QUAY_NAMESPACE}"/"${QUAY_REPOSITORY}"/"${RELEASE}"/helm
+}
 
+#-------------------------------------- Main --------------------------------------#
+delete_csv
```
Comment (gyliu513) [inline common/scripts/push-csv.sh]: @DanielXLee so with this fix, we will only have one CSV in the quay.io for ODLM?

Comment (DanielXLee) [reply common/scripts/push-csv.sh]: yes, when we want to update the CSV, we don't need to delete the whole ODLM application

Comment (gyliu513) [reply common/scripts/push-csv.sh]: Then do we need some flags or options here to disable the `delete_csv`?

Comment (DanielXLee) [reply common/scripts/push-csv.sh]: @gyliu513 we don't need a flag for `delete_csv`, I added this feature just to remove the old CSV from quay.io.

**Pull request merged**